src/app/modules/main/wallet_section/saved_addresses/module.nim

```diff
@@ -42,6 +40,13 @@ method load*(self: Module) =
     ))
   )
 
+method load*(self: Module) =
+  singletonInstance.engine.setRootContextProperty("walletSectionSavedAddresses", newQVariant(self.view))
+
+  self.events.on("savedAddress/changed") do(e:Args):
```

Comment:
You can create a `const` for the event name in the service and use it here

Reply:
Updated